command/agent/config/config.go

```diff
@@ -285,7 +285,9 @@ func LoadConfig(path string) (*Config, error) {
 		case "auto-auth":
 			result.DisableIdleConnsAutoAuth = true
 		case "caching":
-			result.DisableIdleConnsCaching = true
+			result.DisableIdleConnsAPIProxy = true
```

Comment:
Not sure if this is just a style preference, but I recently saw some switch statements where multiple cases had the same block like:

```
case "caching", "proxying": 
    result.DisableIdleConnsAPIProxy = true
```

Not sure if that's something we'd want to follow, or if it makes it easier to miss the 2nd case on the line. Anyway, just worth a mention. Up to you either way is fine by me. 😄

Reply:
Thanks! Adjusted - I think it makes it clearer!